app/controllers/delete_featured_researchers_controller.rb

```diff
@@ -0,0 +1,22 @@
+class DeleteFeaturedResearchersController < ApplicationController
```

Comment:
Would this be better off extending the ContentBlock model/controller from Sufia than it's own controller? Either way, I don't think 'Delete' should be part of the controller name.